plugins/woocommerce-blocks/tests/e2e/plugins/order-confirmation-filters.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Plugin Name: WooCommerce Order Confirmation Filters
```

Comment:
Since these are WC Blocks tests, let's use the `WooCommerce Blocks *` prefix as we do in other test plugins.